concent_api/core/tests/test_integration_subtask_results_verify.py

```diff
@@ -375,11 +374,25 @@ def _create_serialized_subtask_results_verify(
 
     def _create_report_computed_task(self):
         time_str = "2018-04-01 10:00:00"
-        task_to_compute = self._get_deserialized_task_to_compute(
-            timestamp=time_str,
+        compute_task_def = self._get_deserialized_compute_task_def(
             deadline=self._add_time_offset_to_date(time_str, 3600),
             task_id=self.task_id,
-            subtask_id=self.subtask_id
+            subtask_id=self.subtask_id,
+            extra_data={
+                'end_task': 6,
+                'frames': [1],
+                'outfilebasename': 'Heli-cycles(3)',
+                'output_format': BlenderSubtaskDefinition.OutputFormat.JPG.name,  # pylint: disable=no-member
+                'path_root': '/home/dariusz/Documents/tasks/resources',
+                'scene_file': 'blender/scene/ef0dc1/ef0dc1.zzz523',
```

Comment:
Please add helper for creating this path from `task_id` and `subtask_id`, as we be probably reusing it.